src/qtt/algorithms/coulomb.py

```diff
@@ -659,8 +656,8 @@ def findBestSlope(x, y, minimal_derivative=None, fig=None, verbose=1):
         slopes (...)
         results (object): additional data
     """
-    lowvalue = np.percentile(y, 1)
-    highvalue = np.percentile(y, 99)
+    lowvalue, highvalue = np.percentile(y, [5, 99])
```

Comment:
Are you aware you changed the lowvalue to 5th percentile instead of the first?

Reply:
Good spot